mungegithub/mungers/lgtm_handler.go

```diff
@@ -83,7 +98,46 @@ func (LGTMAfterCommitMunger) Munge(obj *github.MungeObject) {
 	}
 }
 
-func (LGTMAfterCommitMunger) isStaleComment(obj *github.MungeObject, comment *githubapi.IssueComment) bool {
+func hasValidReviwer(reviewers []*githubapi.User) bool {
+	for _, r := range reviewers {
+		if r != nil && r.Login != nil {
+			return true
+		}
+	}
+	return false
+}
+
+func foundLGTMFromReviewer(comments []*githubapi.IssueComment, reviewers []*githubapi.User, lastModified *time.Time) bool {
+	for _, c := range comments {
+		if lastModified == nil || c.CreatedAt == nil || (*lastModified).After(*c.CreatedAt) {
+			continue
+		}
+		if !isReviewer(c.User, reviewers) {
+			continue
+		}
+		if strings.ToLower(strings.Trim(*c.Body, " ")) == "lgtm" {
```

Comment:
after some thought, probably equal to is better. sometimes, we say lgtm after XXX to tell author that we are getting really close.